unit_testing/cli/test_cli_sct_detect_pmj.py

```diff
@@ -1,23 +1,32 @@
-from pytest_console_scripts import script_runner
+import math
 import pytest
 import logging
-import os
+
+import numpy as np
+
+from spinalcordtoolbox.image import Image
 from spinalcordtoolbox.scripts import sct_detect_pmj
 
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.script_launch_mode('subprocess')
-def test_sct_detect_pmj_backwards_compat(script_runner):
-    ret = script_runner.run('sct_testing', '--function', 'sct_detect_pmj')
-    logger.debug(f"{ret.stdout}")
-    logger.debug(f"{ret.stderr}")
-    assert ret.success
-    assert ret.stderr == ''
-
-def test_sct_detect_pmj_o_flag(tmp_path):
```

Comment:
where did this test go?

Reply:
Ah, thank you for reminding me! 

I have combined this test with the other test in ee44471. (Since the old test was just using checking if `fname_out` exists, and the new test has to open `Image(fname_out),` it does basically the same thing.)

Reply:
Ah wait, my apologies, there is a slight difference. I will need to make another change.

Reply:
Ah... interesting. I looked into the `sct_detect_pmj` script, and I believe `-igt` is not actually used anywhere in the script! (For context, it was added quite some time ago in https://github.com/neuropoly/spinalcordtoolbox/pull/1453#discussion_r627384597.)

I will open a separate issue for this.

Reply:
Done in https://github.com/neuropoly/spinalcordtoolbox/issues/3377.